src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingVisitor.cs

```diff
@@ -60,9 +60,12 @@ public override void VisitCSharpCodeBlock(CSharpCodeBlockSyntax node)
                 node.Parent is CSharpImplicitExpressionBodySyntax ||
                 node.Parent is RazorDirectiveBodySyntax ||
                 (_currentBlockKind == FormattingBlockKind.Directive &&
-                node.Children.Count == 1 &&
-                node.Children[0] is CSharpStatementLiteralSyntax))
+                node.Parent?.Parent is RazorDirectiveBodySyntax))
```

Comment:
Not super proud of this `if` block here. I didn't want to change this too much because I couldn't recall why some of these existed (I added this 6 months ago without comments. My bad). I've left a comment but I want to clean this up soon.